Tools/CISupport/ews-build/steps.py

```diff
@@ -2521,7 +2522,7 @@ def getQueueStatusFromList(self, change_id, pr_number):
             self.setProperty('pending_prs', pending_prs)
             yield self._addToLog('stdio', 'Required checks are not completed. Waiting until all checks are completed before relabelling PR.\n')
             yield self._addToLog('stdio', f'Missing the following checks: {missing_checks}\n')
-            defer.returnValue(False)
+            defer.returnValue(2)
```

Comment:
Can you name these variables something? It's hard to understand what you expect them to mean (since it's clearly not a normal program status code)